resources/homegrown/data-clinic-for-general.qmd

```diff
@@ -1,76 +1,268 @@
+3. Provide that data for the Requestor as quickly as possible, in the form that the Requestor can use.  
+
+We hypothesise that this benefits both clinical staff requesting the data and the BI team. 
+
+Clinical staff:   
+
+ - Makes the process of getting data from EHRS clearer    
+ - Enhances understanding of the data that is available from EHRS & how it will be useful for their project aims  
+ - Ensures the data provided is in a format that is useful to the Requestor and sensitive to their data analytic skills (e.g. there is no point providing a 10,000 row spreadsheet to someone who struggles to use Excel)     
+ - Overall makes it easier to get the data needed to complete audit/QIP/service evaluations (thereby increasing the number of projects completed, departmental performance and eventually improving patient care)    
+
+BI team:  
+
+  - Ensures the data requests being recieved are realistic and relevant to achieving the aims of projects   
+  - Through the Data Fellow, provides a "go-to" individual for questions about clinical queries  
+  - Better understanding of clinical relevance of their work  
+  - Saves time/frustration emailing back-and-forth with clinical staff  
+
+
+# How do you measure the success of the Data Clinic? 
+
+
+There are multiple metrics that can be used to define "success". What you choose may reflect your department's goals. Examples include: speed of delivery of data, number of projects completed, complexity or projects completed, etc.   
+
+Our two main recommendations from our experience are:  
+
+1. Use a mixed methods approach, collecting both quantitative and qualitative data.     
+    i) The full impact of the service is unlikely to be captured by a single metric - or by just qualitative vs quantitative feedback. Using a mixed methods approach has allowed us to capture a more complete picture of the Data Clinic's impact.   
+2. Collect Outcome, Process and Balancing measures. Similarly to the above, it shall allow you gather a more complete picture of the Data Clinic's impact.  
+    i) Outcome measures capture the Data Clinic's impact on your main aims of the project (i.e. How do you define success?)   
+    ii) Process measures capture the impact of the processes that you are using to try to achieve your goals (i.e. How are you achieving our success?)  
+    iii) Balancing measures capture any unintended consequences of your project, which may offset the benefits it is bringing (e.g. You may be delivering data quickly, but is it of sufficient quality? Are the BI team overworked?)  
+    
+Listed below is our strategy for measuring the impact of the Data Clinic & the reasons behind this approach. This can be copied directly, or adapted to your local environment. We chose 6 key measurements to focus on collecting to avoid overburdening the team. In reality, it may be possible to collect more than this. 
+
+Outcome measures:  
```

Comment:
For this section (here until line 107) maybe underline or make bold these headings?